fluid/DeepASR/tools/profile.py

```diff
@@ -125,7 +125,8 @@ def profile(args):
         class_num=1749,
         parallel=args.parallel)
 
-    adam_optimizer = fluid.optimizer.Adam(learning_rate=args.learning_rate)
+    adam_optimizer = fluid.optimizer.Momentum(
```

Comment:
Please change `adam_optimizer` to `momentum_optimizer`

Reply:
Done. Thx!

Reply:
Done. Thanks!